blocks/init/components/button/button.php

```diff
@@ -8,15 +8,25 @@
 
 namespace Eightshift_Boilerplate\Blocks;
 
-$url   = $attributes['url'] ?? '';
-$title = $attributes['title'] ?? '';
-$id    = $attributes['id'] ?? '';
+$button = $attributes['button'] ?? [];
+
+if ( empty( $button ) ) {
+  return;
+}
+
+$url   = $button['url'] ?? '';
+$title = $button['title'] ?? '';
```

Comment:
I allways get this rule with our rules
Overriding WordPress globals is prohibited. Found assignment to $title

Reply:
yes this is a phpcs linter, we will add a rule to boilerplate